semgrep_output_v1.atd

```diff
@@ -1051,6 +1051,8 @@ type features = {
    ~autofix: bool;
    ~deepsemgrep: bool;
    ~dependency_query: bool;
+   (* since semgrep 1.74.0 *)
+   ~use_batch_upload: bool;
```

Comment:
Does this need to be optional for backwards for compatibility reasons? Just double checking, I think the answer is no.

Reply:
Yes any change to semgrep_output_v1.atd but support backward compatibility; one of the CI check above actually enforces that. The use of ~xxx support backward compability because if the field is not present, it is assumed
to have a default value (I think for boolean the default is false).

Reply:
Good catch. It doesn't strictly need to be optional, but it feels like a good practice. Updated in latest commit